pkg/scheduler/framework/session.go

```diff
@@ -437,6 +438,41 @@ func (ssn *Session) Allocate(task *api.TaskInfo, nodeInfo *api.NodeInfo) (err er
 	return nil
 }
 
+// Unallocate is called when the prebind function fails
+func (ssn *Session) Unallocate(task *api.TaskInfo) error {
```

Comment:
The logic of the Unallocate method is to modify the cache in ssn
1. Change the task status to pending
2. Delete this task (Pod) in the node
3. send Deallocate event (I'm not sure if this step is necessary)
4. set task.Name = = ""